zipline/data/minute_bars.py

```diff
@@ -415,11 +413,49 @@ def write(self, sid, df):
                 volume : float64|int64
             index : DatetimeIndex of market minutes.
         """
+        cols = {
+            'open': df.open.values,
+            'high': df.high.values,
+            'low': df.low.values,
+            'close': df.close.values,
+            'volume': df.volume.values,
+        }
+        dts = df.index.values
```

Comment:
@richafrank is this one of the perf regressions you mentioned in pandas v.recent?